src/imex/imex.interface.ts

```diff
@@ -77,8 +85,28 @@ export interface IMEXOptions {
   ignoreEmpty?: boolean
 }
 
-export type IMEXColumn<D extends object = any> = Column<D> & {
+export type IMEXColumn<D extends object = any> = Omit<
+  Column<D>,
+  'Header' | 'accessor'
+> & {
   imex?: IMEXOptions
-}
+} & (
+    | {
+        Header: string
+      }
+    | {
+        Header?: Exclude<Column<D>['Header'], 'string'>
```

Comment:
```suggestion
        Header?: Exclude<Column<D>['Header'], string>
```
Isn't it? If so, don't forget to do the same for `accessor`.

Reply:
Could you add some type assertions* at the end of this file, to make sure we don't break it in the future?

*You know with the directive `@ts-expect-error` :wink: